frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -867,10 +867,22 @@ function ReaderRolling:onUpdatePos()
         -- we have set above) to avoid multiple refreshes.
         return true
     end
+
+    if Device:isAndroid() then
+        UIManager:discardEvents(60)
+        Device:setIgnoreInput(true)
+    end
+
     -- Calling this now ensures the re-rendering is done by crengine
     -- so updatePos() has good info and can reposition
     -- the previous xpointer accurately:
     self.ui.document:getCurrentPos()
+
+    if Device:isAndroid() then
+        Device:setIgnoreInput(false)
+        UIManager:discardEvents(true)
+    end
```

Comment:
And may be this one is enough ?
If not, try it before setIgnoreInput(false).

There could also be a small issue with @NiLuJe ultra tuning of UIManager by reusing `self._now` (which might be 20 seconds ago if the re-rendering took 20s) instead of an updated `TimeVal:now()`.
You could try updating:
https://github.com/koreader/koreader/blob/d9730cf0fc15647c637c6ba4661cabd60c084733/frontend/ui/uimanager.lua#L1084
to be `self._discard_events_till = TimeVal:now() + delay`

Reply:
If I put the unchanged `discardEvents` before `setIgnoreInput`, then taps during rerendering are executed immediately after the rendering (which is bad). 
If I put the unchanged `discardEvents` after `setIgnnoreInput` the behavior is as expected.